addons/l10n_cn_small_business/data/account.account.template.csv

```diff
@@ -1,76 +1,206 @@
-"id","name","code","user_type_id/id","chart_template_id/id","reconcile"
-"l10n_cn_1012","Other Monetary Funds","1012","account.data_account_type_current_assets","l10n_cn_small_business.l10n_chart_china_small_business","False"
```

Comment:
It is of course logical that the CoA is in the language of the country itself, but it is quite a heavy change for stable.  On the other hand, the l10n_cn module is multilang, so you could add the translation to English.  (I suppose there is a reason it was originally in English)